src/style.css

```diff
@@ -84,10 +87,31 @@ button i {
   flex-basis: 95%;
 }
 
-.list-content span {
+.todo-description {
   flex-basis: 95%;
+  height: 2rem;
+  border: none;
+  outline: none;
+  font-size: 1rem;
+  background: none;
 }
 
 svg {
   fill: #eee;
 }
+
+.strike {
+  text-decoration: line-through;
+}
+
+.disabled {
+  opacity: 0.5;
+}
+
+/* .disabled .todo-description {
+  pointer-events: none;
+} */
+
+/* .disabled .todo-menu {
+  pointer-events: none;
+} */
```

Comment:
- [ ] Kindly remove all codes that are commented out in your project. Removing such codes helps keep your code clean.

Reply:
I fixed it @Gambit142 :+1: 
Thanks for reviewing my project :+1: